src/org/rascalmpl/library/lang/rascal/tests/library/IO.rsc

```diff
@@ -25,3 +26,16 @@ test bool createdDoesNotCrashOnURIRewrites() {
     writeFile(|tmp:///createdDoesNotCrashOnURIRewrites/someFile.txt|, "123456789");
     return created(|tmp:///createdDoesNotCrashOnURIRewrites/someFile.txt|) <= now();
 }
+
+test bool testWriteBase32() {
+    str original = "Hello World!";
+    writeBase32(|tmp:///base32Test/writeTest.txt|, toBase32(original));
+    return original == readFile(|tmp:///base32Test/writeTest.txt|);
```

Comment:
isn't there a `fromBase32`  missing here?

Reply:
No, the "writeBase32" does not write base32 to file. It takes a base32 encoded string and writes the decoded bytes to file. The resulting file should therefore contain the original (unencoded) string.